From 5a9e39508d5b81ea70ba0461d677c65e179fd561 Mon Sep 17 00:00:00 2001
From: Colin Xu <colin.xu@intel.com>
Date: Wed, 18 Nov 2020 13:22:24 +0800
Subject: [PATCH] VM_MANAGER: Remove disable_idle_d3 start_civ.sh

No need to add disable_idle_d3=1 to workaround the
guest intel-lpss driver initialization failure. Use
the new pm_restore_ids so that the specific device
can save/restore device during power transition.
Since pm_restore_ids is already added to kernel
command line which also applies to loadable modules,
no need to add it in start_civ.sh.

Tracked-On: OAM-94624
Change-Id: I6c6d096470177a47eb1012deb4e257b709ed6fae
Signed-off-by: Colin Xu <colin.xu@intel.com>
---
 scripts/start_civ.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/start_civ.sh b/scripts/start_civ.sh
index 2e09d17..4e27b70 100755
--- a/scripts/start_civ.sh
+++ b/scripts/start_civ.sh
@@ -521,7 +521,7 @@ function set_pt_pci_vfio() {
     local PT_PCI=$1
     local unset=$2
     if [ ! -z $PT_PCI ]; then
-        sudo sh -c "modprobe vfio-pci disable_idle_d3=1"
+        sudo sh -c "modprobe vfio-pci"
         local iommu_grp_dev="/sys/bus/pci/devices/$PT_PCI/iommu_group/devices/*"
         local d
         for d in $iommu_grp_dev; do
--
2.17.1

